PAY-744: Order payment status reports (#911)
magento/merchdocs: src/magento-payments/order-payment-status.md

## Download order payment statuses

You can download a .csv file with all of the statuses visible in the Order payment status view grid, whether you are viewing the default 30 days of statuses or a customized timeframe.

1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Order payment status**.
1. If you want to see statuses for a timeframe other than the last 30 days, [customize the date range timeframe for your statuses](#customize-statuses-timeframe).
1. Click the _Download_ (![]({% link images/images/btn-download.png %})) icon.

Your order payment statuses are downloaded in a .csv format.

## Default order payment status timeframes

These order payment status timeframes are currently available in Magento Payments.

Report | Description
------------ | --------------------
Yesterday | Available from the Order payment status dates selector, this shows information for the prior date.
Today | Available from the Order payment status dates selector, this shows information for the current day.
Last 7 days | Available from the Order payment status dates selector, this shows information for the last seven days.

> Last 30 days | Available from the Order payment status dates selector and by default in the Order paymen statuses view, this shows information for the last 30 days.

Reviewer: typo

```suggestion
Last 30 days | Available from the Order payment status dates selector and by default in the Order payment statuses view, this shows information for the last 30 days.
```

Last 90 days | Available from the Order payment status dates selector, this shows information for the last 90 days.
Year to date | Available from the Order payment status dates selector, this shows information for the the entire year to date.
Custom range | Available from the Order payment status dates selector, this can be filtered to show a custom date range.

## Order payment status information

The Order payment status view shows extensive info for each status shown in the grid.

### Column descriptions

Order payment status reports include the following information.

Column | Description
------------ | --------------------

> Order ID | Magento order ID<br> <br>Click the ID to see related [order info](https://docs.magento.com/user-guide/sales/orders.html).

Reviewer: We use relative links for topics within the user guide and not the HTML URL. 

```suggestion
Order ID | Magento order ID<br> <br>Click the ID to see related [order info]({% link sales/orders.md %})
```

Order Date | Order date timestamp
Authorized Date | Date timestamp of payment authorization

> Order Status | Current Magento [order status](https://docs.magento.com/user-guide/sales/order-status.html)

Reviewer: same here
```suggestion
Order Status | Current Magento [order status]({% link sales/order-status.md %})
```
